lib/core/versions/latest/Did.ts

```diff
@@ -63,7 +64,7 @@ export default class Did {
    */
   public static async create (didString: string, didMethodName: string): Promise<Did> {
     const methodNameParts = didMethodName.split(':');
-    if (methodNameParts.length < 2) {
+    if (methodNameParts.length > 2) {
```

Comment:
What if method name is `sidetree:some:thing`? then we don't accept that?

Reply:
I'd say we don't, unless we find a need for it, for now.

Reply:
Thats not valid, please review did core: https://w3c.github.io/did-core/#did-syntax

Reply:
Apparently, this case is allowed... https://github.com/transmute-industries/vc.transmute.world/blob/master/src/services/unlockedDIDs/did:v1:test:nym:z6MkhdmzFu659ZJ4XKj31vtEDmjvsi5yDZG5L7Caz63oP39k.json